### DISCUSSION TITLE
Python-Jose abandoned - use authlib or other instead?

### DISCUSSION THREAD

**Discussion by SpoonOfDoom** [fastapi/full-stack-fastapi-template#1188] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
No example needed
```

### Description

First of all, thank you for all your hard work on FastApi and this template!

I just wanted to raise the discussion here that can also be found over on the FastApi repo, which concerns the usage of Python-Jose: https://github.com/tiangolo/fastapi/discussions/9587

TLDR: it seems that Python-Jose has been abandoned for a while, and now CVEs have been popping up surrounding it and its dependencies. Yet, it is still used in this template and suggested in the official FastApi documentation. Some people are reporting that it doesn't work at all in Python 3.11, although I personally haven't encountered that issue.

Would it maybe be appropriate to replace the Jose dependency in this project with authlib or something else? This project, being a template, should probably be as future proof as possible.

### Operating System

Windows

### Operating System Details

Not relevant to topic

### Python Version

3.11.9

**Comment by estebanx64** [comment, marked as answer]
already fixed and merged
https://github.com/tiangolo/full-stack-fastapi-template/pull/1203

**Comment by dss010101** [comment]
what is the recommended library to use today for authentication with fastapi?
